### ISSUE TITLE
Sdtrig support

### ISSUE THREAD

**Issue by sebastien-riou** [riscv/sail-riscv#222]
If I am not mistaken, Sdtrig extension is not supported yet. Is it planned at some point ?

**Comment by Mudassir10X** [comment]
Yes, it is a work in progress that has long been delayed. I am currently working on it. you can refer to the Draft PR #486
